src/sentry/issues/grouptype.py

```diff
@@ -43,6 +70,62 @@ def __post_init__(self) -> None:
         if self.category not in valid_categories:
             raise ValueError(f"Category must be one of {valid_categories} from GroupCategory.")
 
+        # ensure that noise config ignore limits only increase with rollouts
+        # use a ratio of ignore limit to expiry time to allow for greater flexibility
+        if self.group_policy:
+            group_policy = self.group_policy
+
+            if group_policy.feature:
+                try:
+                    features.get(group_policy.feature, None)
+                except FeatureNotRegistered:
+                    raise ValueError(f"Feature {group_policy.feature} is not registered")
```

Comment:
We might be able to auto register these features using `feature.add`. I'm not sure if we'll need extra work to set a default in `server.py`, but maybe we can change the default feature handler to assume false if it's not defined.

Reply:
Was thinking about this. Will probably be a follow up PR when we can have both noise reduction + auto register features under one roof.